src/components/SnackbarManager.tsx

```diff
@@ -36,8 +36,8 @@ const SnackbarManager = () => {
   useEffect(() => {
     let timer: ReturnType<typeof setTimeout>
 
-    if (message) {
-      timer = setTimeout(() => dispatch(snackbarDisplayTimeExpired()), message.duration || 3000)
+    if (message && message.duration >= 0) {
+      timer = setTimeout(() => dispatch(snackbarDisplayTimeExpired()), message.duration)
```

Comment:
Not directly linked to this PR, but I noticed that snackbar messages appears below modals.

Reply:
Thanks for letting me know, I'll have a look

Reply:
✅ Fixed in https://github.com/alephium/desktop-wallet/pull/593/commits/089d037278e9009ce48735401d9a6259f1bc2734